index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>José Fernández-Caminero: Full-Stack Engineer; Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <nav>
+        <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+        <img src="resources/ic_menu.svg" alt="Toolbar menu">
+    </nav>
+
+    <h1>José Fernández-Caminero</h1>
+
+    <p>I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
+
+    <button>Start collaboration</button>
```

Comment:
- [ ] The text of the button is a little bit smaller than the one in the Figma design please try to make it the same font size 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/170577970-264edbbe-ce78-4ee8-a53d-5b00424f82d5.png)
- Figma
![image](https://user-images.githubusercontent.com/61790587/170577994-10167338-7a4d-4833-b3bc-df7dd750e3ce.png)